webadmin/views.py

```diff
@@ -148,3 +148,19 @@ def add_user(request):
         form = AddUserForm()
     template_data['form'] = form
     return render(request, 'ienitk/admin/createuser.html', template_data)
+
+#View all candidates and their status
+def candidates_view(request):
```

Comment:
If the Account is `Admin`, should we show the status of all the users?
@mehnazyunus @madhumithanara @chaitany10 ?

Reply:
Yes..guess so

Reply:
Yes